tensorflow/python/keras/layers/pooling.py

```diff
@@ -479,6 +479,9 @@ class AveragePooling2D(Pooling2D):
       4D tensor with shape `(batch_size, pooled_rows, pooled_cols, channels)`.
     - If `data_format='channels_first'`:
       4D tensor with shape `(batch_size, channels, pooled_rows, pooled_cols)`.
+  
+  >>>average_pooling_layer = tf.keras.layers.AveragePooling2D(pool_size = (2,2),strides = (1,1),padding = "same")
```

Comment:
Please see [here](https://www.programcreek.com/python/example/89711/keras.layers.AveragePooling2D) for an example of how to use `tf.keras.layers.AveragePooling2D`. You need to call it on a tensor, ie

```
x = tf.keras.Input(shape=(3, 3, 1), name='x')
y = average_pooling_layer = tf.keras.layers.AveragePooling2D(pool_size = (2,2),strides = (1,1),padding = "same")(x)
```

In this example, `y` would return
```
Tensor("average_pooling2d_9/Identity:0", shape=(None, 3, 3, 1), dtype=float32)
```
I hope this makes sense :)

Reply:
Ohhhhhhhhh. ok. does this work with numpy arrays as well? I've only used Sequential models till now ( Even that I just learnt about about a month ago )

Reply:
Hi I tried what you said in colab and it return `<tf.Tensor 'average_pooling2d_2/Identity:0' shape=(None, 1, 1, 1) dtype=float32>`

Reply:
That's odd, the shapes don't even match up. Maybe I'm understanding it wrong, and in the meantime wait for a reviewer with write access looks at this?

Reply:
I don't think `tf.keras.layers.AveragePooling2D` works directly with numpy arrays since it is a layer of a `tf.keras` model, so an input layer is required (I think the docs say that it takes a 4D tensor as an input)

Reply:
I think its to do with the strides and the pool size. I've encountered this before.

Reply:
@WilliamHYZhang I tried it too and got what you got - not sure what's causing the discrepancy

Reply:
> I don't think `tf.keras.layers.AveragePooling2D` works directly with numpy arrays since it is a layer of a `tf.keras` model, so an input layer is required (I think the docs say that it takes a 4D tensor as an input)

ohh. ok

Reply:
I tried it again at https://colab.research.google.com/drive/1X2YlL7CK97JibbwinRh7wXMuxhjJp7mu with different strides and pool sizes and got a goo output. Haha theres an advantage in making so many errors in code. I just have a sense now of whats wrong lol

Reply:
haha trial and error is my favorite, nice job!